ffi/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(unsafe_code)]
+#![allow(unsafe_code, clippy::extra_unused_lifetimes)]
```

Comment:
finally we got this lint, I was always surprised this is valid rust code or at least not deny by default lint

Reply:
you could have fixed this directly in the code instead of allowing the lint, there is only one occurrence triggering this lint in the `impl_tuple`. It's ok to remove the lifetime declaration

Reply:
This lint has had a lot of false-positives, which is why it wasn't enabled by default, and why I disabled it here.